src/components/PersonHoverCard/index.tsx

```diff
@@ -49,37 +44,30 @@ const PersonHoverCard: React.FunctionComponent<{ personId: number }> = ({
 
   if (person) {
     return (
-      <Box onMouseEnter={openPopover} style={{ display: 'flex' }}>
+      <Box
+        onMouseEnter={openPopover}
+        onMouseLeave={closePopover}
+        style={{ display: 'flex' }}
+      >
         {children}
-        <Popover
+        <Popper
```

Comment:
Another nice to have would be a cheeky little fade in animation. Not required of course.